art/detection/subsetscanning/scanningops.py

```diff
@@ -0,0 +1,144 @@
+""" Scanning operations """
+import numpy as np
+
+
+class ScanningOps:
+    """ Specific operations done during scanning  """
+
+    @staticmethod
+    def optimize_in_single_dimension(pvalues, a_max, image_to_node, score_function):
```

Comment:
Could you please add a docstring describing what the function does, its arguments and return similar to the other function docstrings in ART?